datumaro/plugins/ade20k2020_format.py

```diff
@@ -174,6 +175,19 @@ def _load_class_mask(path):
         return mask
 
 class Ade20k2020Importer(Importer):
+    @classmethod
+    def detect(cls, context: FormatDetectionContext) -> None:
+        annot_path = context.require_file('*/**/*.json')
+
+        with context.probe_text_file(
+            annot_path, "must be a JSON object with an \"annotation\" key",
+        ) as f:
+            contents = json.load(f)
+            if not isinstance(contents, dict):
+                raise Exception
+            if 'annotation' not in contents:
+                raise Exception
```

Comment:
Looks pretty similar to `datumaro` and `coco`. How the conflict is resolved?

Reply:
In this format, the key is `annotation`. In `coco`, it's `annotations`.

In Datumaro it's neither, so I don't think it's even similar. 🙂